packages/target-tools/src/telemetryProvider.spec.js

```diff
@@ -28,22 +27,12 @@ describe("TelemetryProvider", () => {
         }
       })
     );
+    expect(mockExecute.mock.calls[0][1][0].execution).toBe(1);
```

Comment:
Minor nitpick for sake of consistency - let's remove this line and add it into the previous assertion:
`expect.objectContaining({........,
    execution: 1
})`